lib/cisco_node_utils/cmd_ref/vxlan_vtep.yaml

```diff
@@ -1,5 +1,7 @@
 # vxlan_vtep
 ---
+_exclude: [/N(3|5|6)/]
```

Comment:
You should be able to remove the n3k reference here now right?

https://github.com/cisco/cisco-network-node-utils/blob/feature/vxlan-vtep-vni-test-fix/lib/cisco_node_utils/cmd_ref/vxlan_vtep.yaml#L32